src/scss/components/eyebrow/index.scss

```diff
@@ -0,0 +1,126 @@
+// ----- BASE ELEMENTS ---------------
+
+.eyebrow {
+  overflow: hidden;
+  width: 100%;
+  height: 56px;
+  background-color: #000000;
+}
+
+.img-left {
+  float: left;
+  width: 159px;
+  height: 56px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.right-menu {
+  float: right;
+}
+
+.eyebrow a img {
+  width: 119px;
+  height: 24px;
+}
+
+.eyebrow a {
+  float: left;
+  font-size: 13px;
+  color: #ffffff;
+  text-align: center;
+  padding: 1rem 9px;
+  text-decoration: none;
+  line-height: 1;
+  height: 56px;
+}
+
+.eyebrow p {
+  color: #ffffff;
+  line-height: 1.5;
+}
+
+.dropdown {
+  float: right;
+  overflow: hidden;
+}
+
+.dropdown .dropbtn {
+  font-size: 12px;
+  border: none;
+  outline: none;
+  color: #ffffff;
+  padding: 14px 16px;
+  background-color: inherit;
+  font-family: inherit;
+  margin: 0;
+  height: 56px;
+}
+
+.eyebrow a:hover, .dropdown:hover .dropbtn {
+  background-color: #242424;
+}
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #242424;
+  min-width: 125px;
+  z-index: 1;
+}
+
+.dropdown-content a {
+  float: none;
+  color: #ffffff !important;
+  padding: 12px 10px;
+  text-decoration: none;
+  display: block;
+  text-align: left;
+}
+
+.dropdown-content a:hover {
+  background-color: #242424 !important;
+  color: #e6dddd !important;
+}
+
+.dropdown:hover .dropdown-content {
+  display: block;
+}
+
+.eyebrow hx-icon {
+  color: #ffffff;
+  font-size: 15px;
+}
+
+.eyebrow-menu {
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+  display: inline-block;
+  font: inherit;
+  font-weight: 500;
+  line-height: 1;
+  border-radius: 2px;
+  justify-content: center;
+  font-size: 0.875rem;
+  padding: 22px 2rem;
+  color: #f5f5f5;
+}
+
+#eyebrowMenuId, .eyebrow-menu:hover, #eyebrowMenuId hx-menuitem:hover {
+  background-color: #242424 !important;
+  color: #e6dddd !important;
+}
+
+#eyebrowMenuId {
```

Comment:
Please refactor.

We should be able to use our component styles rather than id's to style the page.

If we need specific styling for the demo page, let's discuss.